### ISSUE TITLE
unit_label missing from ProductCreateOptions

### ISSUE THREAD

**Issue by randyjeppesen** [stripe/stripe-dotnet#1363]
When creating or updating a product, UnitLabel is missing from the ProductCreateOptions and ProductUpdateOptions.  I can't update this property through code, only through the Stripe Dashboard.

**Comment by karlr-stripe** [comment]
Hi @randyjeppesen — you're correct, that property is missing, thank you for reporting it! We'll work to add it, but for now it is possible to include this property(or any other one that is not mapped in the library) via `ExtraParams` like this : 

```
var service = new ProductService();
Product product = service.Create(new ProductCreateOptions
{
   Name = "Monthly membership base fee",
   Type = "service",
   ExtraParams = new Dictionary<string, string> {
       {"unit_label" , "seats"}
   }
});
```

**Comment by ob-stripe** [comment]
Fixed in 20.3.0.
